hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeysDeleteRequest.java

```diff
@@ -215,20 +222,22 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
     }
 
     addDeletedKeys(auditMap, deleteKeys, unDeletedKeys.getKeysList());
-
+    long endNanos = Time.monotonicNowNanos();
```

Comment:
Again, this can be added within the `finally` block or you may also have separate `endNanos` for both success and failure latencies (adjacent to `Result.Success`/`Result.Failure`).